src/main/java/org/sagebionetworks/web/client/widget/discussion/DiscussionThreadWidgetViewImpl.java

```diff
@@ -147,11 +147,6 @@ public void setMessage(String message) {
 	}
 
 	@Override
-	public void setActiveUsers(String activeUsers){
```

Comment:
I would expect a change to view.clear() (to remove all from author and activeUsers containers).